docs/install.md

```diff
@@ -682,6 +789,103 @@ You can modify PMEM-CSI to support more xxx by changing the `variable` from Y to
+| pullPolicy | string | Docker image pull policy. either one of `Always`, `Never`, `IfNotPresent` | `IfNotPresent` |
+| logLevel | integer | PMEM-CSI driver logging level | 3 |
+| deviceMode | string | Device management mode to use. Supports one of `lvm` or `direct` | `lvm`
+| controllerResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for controller pod |
+| nodeResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for the pods running on node(s) |
+| registryCert | string | Encoded tls certificate signed by a certificate authority used for driver's controller registry server | generated by operator self-signed CA |
+| nodeControllerCert | string | Encoded tls certificate signed by a certificate authority used for driver's node controllers | generated by operator self-signed CA |
+| registryKey | string | Encoded RSA private key used for signing by `registryCert` | generated by the operator |
+| nodeControllerKey | string | Encoded RSA private key used for signing by `nodeControllerCert` | generated by the operator |
+| caCert | string | Certificate of the CA by which the `registryCert` and `controllerCert` are signed | self-signed certificate generated by the operator |
+| nodeSelector | string map | [Labels to use for selecting Nodes](../docs/install.md#run-pmem-csi-on-kubernetes) on which PMEM-CSI driver should run. | `{ "storage": "pmem" }`|
+| pmemPercentage | integer | Percentage of PMEM space to be used by the driver on each node. This is only valid for a driver deployed in `lvm` mode. This field can be modified, but by that time the old value may have been used already. Reducing the percentage is not supported. | 100 |
+| labels | string map | Additional labels for all objects created by the operator. Can be modified after the initial creation, but removed labels will not be removed from existing objects because the operator cannot know which labels it needs to remove and which it has to leave in place. |
+
+<sup>1</sup> To use the same container image as default driver image
+the operator pod must set with below environment variables with
+appropriate values:
+ - POD_NAME: Name of the operator pod. Namespace of the pod could be figured out by the operator.
+ - OPERATOR_NAME: Name of the operator container. If not set, defaults to "pmem-csi-operator"
+
+<sup>2</sup> Image versions depend on the Kubernetes release. The
+operator dynamically chooses suitable image versions. Users have to
+take care of that themselves when overriding the values.
+
+**WARNING**: although all fields can be modified and changes will be
```

Comment:
This warning can be removed, right?

Reply:
Node stage and publish still might not work. I mean:
 - start the driver in `lvm`
 - Create volume
 - Run application using that volume
 - switch driver mode to `ndctl`
 - restart the application

This supposed to fail.

Reply:
I've reopened https://github.com/intel/pmem-csi/issues/632 because of this.